hack/lib/version.sh

```diff
@@ -142,6 +142,7 @@ kube::version::ldflag() {
   # If you update these, also update the list pkg/version/def.bzl.
   echo "-X '${KUBE_GO_PACKAGE}/pkg/version.${key}=${val}'"
   echo "-X '${KUBE_GO_PACKAGE}/vendor/k8s.io/client-go/pkg/version.${key}=${val}'"
+  echo "-X 'k8s.io/kubernetes/pkg/kubectl/version.${key}=${val}'"
```

Comment:
nit: Could we also use `${KUBE_GO_PACKAGE}` here?

Reply:
Yes please.